src/main/java/uk/gov/companieshouse/api/accounts/service/AbstractParentService.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.companieshouse.api.accounts.service;
+
+import uk.gov.companieshouse.api.accounts.LinkType;
+import uk.gov.companieshouse.api.accounts.exception.DataException;
+import uk.gov.companieshouse.api.accounts.model.rest.RestObject;
+
+public interface AbstractParentService<T extends RestObject> extends AbstractService<T> {
```

Comment:
The class name "AbstractParentService" suggests that this is an abstract class. I think the interface should be named what it is, a ParentService with a JavaDoc explaining what the interface is a contract for (it should be clear what the difference is between a service and a parent service without having to go looking at the code).

Reply:
Agreed I will rename them.